### PR TITLE
{math}[intel/2015a] Octave 3.8.2 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5332d2bfe6d375d3e5bd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e367b131a22c4e3ed876 for a full test report.

**Comment by fgeorgatos** [comment]
Hi there,

not to spoil the fun, but back then I recall doing `ldd` and finding some missing deps, see:
https://github.com/hpcugent/easybuild-easyconfigs/pull/196/files
Of course, missing `--with-blas` was likely an omission in my PR.

**Comment by boegel** [comment]
@fgeorgatos: although this is definitely not a complete Octave, it seems like a good start; more dependencies can be added later. The major trick here is the patch included to get Octave to build with Intel compilers, that was certainly non-trivial...

File: easybuild/easyconfigs/o/Octave/Octave-3.8.2-foss-2015a.eb [hunk 1]
```diff
@@ -11,17 +11,19 @@ toolchain = {'name': 'foss', 'version': '2015a'}
...
 
-configopts = '--with-blas="${LIBBLAS}"'
-
 dependencies = [
     ('PCRE', '8.36'),
     ('ncurses', '5.9'),
     ('libreadline', '6.3'),
 ]
 
+configopts = '--with-blas="${LIBBLAS}"'
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/Octave/Octave-3.8.2-foss-2015a.eb]: why deviate from our usual style (the {} )?

Comment (boegel) [inline easybuild/easyconfigs/o/Octave/Octave-3.8.2-foss-2015a.eb]: I don't think there's a reason, I'll change it.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/abeb1783a791d783fa76 for a full test report.

**Comment by boegel** [comment]
(still) works like a charm, so going in; it can be enhanced with more dependencies in the future

Thanks for the review @fgeorgatos and @wpoely86!

**Pull request merged**